.github/workflows/deploy.yml

```diff
@@ -37,6 +49,7 @@ jobs:
       - name: Deploy to GitHub Pages
         uses: peaceiris/actions-gh-pages@v3
         with:
+          if: ${{ github.ref == 'refs/heads/main' }}
```

Comment:
does it come here or above?

Reply:
see https://github.com/microsoft/jacdac-docs/blob/main/.github/workflows/build.yml#L30

Reply:
I've cancelled the workfull. the 'if' needs to be same level as 'uses' @rlisahuang